shell/platform/android/io/flutter/plugin/platform/PlatformPlugin.java

```diff
@@ -240,105 +241,98 @@ public void onSystemUiVisibilityChange(int visibility) {
   private void setSystemChromeEnabledSystemUIMode(PlatformChannel.SystemUiMode systemUiMode) {
     int enabledOverlays;
 
-    if (systemUiMode == PlatformChannel.SystemUiMode.LEAN_BACK
-        && Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {
+    Window window = activity.getWindow();
+    View view = window.getDecorView();
+    WindowInsetsControllerCompat windowInsetsControllerCompat =
+        new WindowInsetsControllerCompat(window, view);
+
+    if (systemUiMode == PlatformChannel.SystemUiMode.LEAN_BACK) {
       // LEAN BACK
-      // Available starting at SDK 16
+      // Available starting at SDK 20, due to the backwards compatibility provided by the
```

Comment:
It would be great if we can eventually migrate from the deprecated APIs to use AndroidX's compat libraries, but according to https://docs.flutter.dev/development/tools/sdk/release-notes/supported-platforms we are also supporting API 19. Seems like everything here and below will require API 20+. We might need to rethink a bit on the supported modes and the default mode (especially if edge-to-edge, which works best on API 29+, is going to be the new default).